instrumentation/aws-sdk/aws-sdk-1.11/src/main/java/io/opentelemetry/auto/instrumentation/awssdk/v1_11/AWSHttpClientInstrumentation.java

```diff
@@ -74,14 +74,12 @@ public static void methodExit(
         @Advice.Argument(value = 0, optional = true) final Request<?> request,
         @Advice.Thrown final Throwable throwable) {
       if (throwable != null) {
-        SpanWithScope spanWithScope = request.getHandlerContext(SPAN_SCOPE_PAIR_CONTEXT_KEY);
-        if (spanWithScope != null) {
+        SpanWithScope scope = request.getHandlerContext(SPAN_SCOPE_PAIR_CONTEXT_KEY);
+        if (scope != null) {
           request.addHandlerContext(SPAN_SCOPE_PAIR_CONTEXT_KEY, null);
-          Span span = spanWithScope.getSpan();
-          DECORATE.onError(span, throwable);
-          DECORATE.beforeFinish(span);
-          span.end();
-          spanWithScope.closeScope();
+          Span span = scope.getSpan();
```

Comment:
I did try to replace SpanWithScope with Scope, but I have a problem getting the span out of the Scope.  
I need some feedback on howto.

Reply:
i'm not sure what to do about this either, i recommend out of scope for this PR

Reply:
Not sure how to setup onMethodEnter... so that we can take local "otelSpan" as parameter and then onMethodExit can pass that span as an incoming parameter.. that's the pattern we have elsewhere.  @anuraaga might be able to help with this?